pkg/image/importer/client.go

```diff
@@ -151,7 +151,14 @@ func (r *repositoryRetriever) ping(registry url.URL, insecure bool, transport ht
 	versions := auth.APIVersions(resp, "Docker-Distribution-API-Version")
 	if len(versions) == 0 {
 		glog.V(5).Infof("Registry responded to v2 Docker endpoint, but has no header for Docker Distribution %s: %d, %#v", req.URL, resp.StatusCode, resp.Header)
-		return nil, &ErrNotV2Registry{Registry: registry.String()}
+		switch {
+		case resp.StatusCode >= http.StatusOK && resp.StatusCode < http.StatusMultipleChoices:
```

Comment:
Using named constant there hides the intention (status >= 200 && status <= 299).